servicemesh-install/topics/install-remove.adoc

```diff
@@ -1,19 +1,48 @@
-[[removing_service-mesh]]
+[[removing-service-mesh]]
 = Removing {ProductName}
 
-The following command removes the {ProductShortName} from an existing installation. Log in as a cluster admin and run these commands from any host with access to the cluster.
+The following process removes the {ProductShortName} from an existing installation. Log in as a cluster admin and run these commands from any host with access to the cluster.
 
-. Remove the custom resource by running the following command:
+[[remove-control-plane]]
+== Remove the control plane
+
+. Retrieve the name of the installed custom resource with this command:
 +
 
 ----
-$ oc delete -n istio-operator installation istio-installation
+oc get controlplanes -n istio-system
 ----
 
 +
-. Remove the operator by running the following command:
-+
+. Remove the custom resource by running the following command:
+
+----
+$ oc delete -n istio-system -f <name_of_custom_resource>
+----
+
+[NOTE]
+====
+When you remove the custom resource, {ProductShortName} tells the operator to begin uninstalling everything it installed.
+====
+
+
+[[remove-operator]]
+== Remove the operator
+Remove the operator by running the following commands:
+
+----
+$ oc delete -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.10/deploy/istio_v1alpha1_installation_crd.yaml
```

Comment:
@brian-avery, I was thinking that (especially in the remove operation) that you would be referencing files that had been copied to the local server, not pulling from somewhere else?  Because you'd want to be removing the version that you currently had installed, wouldn't you?

Reply:
Can @knrc and @tvieira possibly weigh in on how we want to handle this?

Reply:
@geekspertise Do we have a practice of having an appendix to write yaml file examples (i.e. link to appendix C for CRD examples)? I don't like the idea of pointing to Github. We have a moving branch practice and, for the current link, the file already moved to another folder and this link is `404` now. 

Ideally, an appendix with a couple of examples within the document is the best I can think of, so someone can copy and paste within a file and use it as a reference. The additional work with having yaml files in appendix is that we have to test it before pushing changes live, but I'm good adding that to the QE task list.

Reply:
@tvieira We could place yaml file examples within our documentation set. I would prefer that we had control of the files rather than hosting them elsewhere and serving up bad links, so I agree with you.

Reply:
The yaml files were condensed down to one servicemesh-operator.yaml.